docs/user/collaboration.rst

```diff
@@ -0,0 +1,85 @@
+.. _collaborating:
+
+Collaborating on Renku
+======================
+
+Beyond building and maintaining your own data science workflows, there are many
+ways to use Renku to share your project with collaborators.
+
+Finding & accessing public projects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To find and access projects on the Renku platform that you might want to
+interact with, you can click the **Projects** link in the menu bar followed by
+the **Search** tab. Without entering any search terms, you will see a list of
+all projects that you have access to. You can click on a project name to be
+taken to the project's landing page.
+
+.. warning::
+
+  Note that the search box currently only allows you to search by project title,
+  not by username.
+
+By default, the files in projects created by other people are "read only" to you.
+This means you can see an overview, create and read Kus, and browse the filesystem
+in both text and :ref:`lineage` modes. The project's **Notebook Servers** tab
+will inform you that you do not have permissions to launch a notebook or make
+modifications to the code or datasets.
+
+If you would like to interact with the notebooks (run, edit, upload data, continue
+your own analyses, etc.), you can `fork the project <forks_for_collaboration_>`_,
+which creates your own copy of the current state of the repo.
+
+If you have questions, notes, or discussion topics that you would communicate to
+the project's owner(s), you can make `Renku platform "Ku"s <make_kus_>`_
+
+If you are part of the team and want to work on this project, you can ask the
+project creator to `add you as a developer <added_to_project_>`_ or maintainer
+of the project through GitLab.
+
+.. _forks_for_collaboration:
+
+Fork a project and take it in your own direction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Forking someone else's project on Renku is just like forking any version-controlled
+project. The Renku platform provides you a fork button on the project's home page
+that creates a fork of the project (with the same name) under your own namespace,
+where you have permissions to launch notebooks and make settings modifications.
+
+One reason you might want to fork someone's project is to use it as a template for
+your own analysis.
+
+.. warning::
+
+  Note that you will be unable to fork a project that has an identical name to a
+  project already in **Your Projects** list. We suggest you change the name of
+  your own project (which you can do in the Renku interface) before forking.
+
+.. _make_kus:
+
+Add collaborators to the conversation through "Ku"s
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In Renku, Kus are media-rich discussions you can use to help keep track of
+your work and to collaborate with others. To find out more about them and walk
+through sample usage, see :ref:`sharing_is_caring`.
+
+.. _added_to_project:
+
+Add or be added as a developer on a project
```

Comment:
A bit late to the party but... found a typo here

Reply:
oh no! where?! I don't see it!

Reply:
oh sorry, I accidentally modified the code in the local repo, but here everything seems fine.
My bad :man_facepalming: